librad/src/net/gossip.rs

```diff
@@ -848,7 +846,7 @@ where
     async fn broadcast<'a, M, X>(&self, rpc: M, excluding: X)
```

Comment:
No complaint from clippy about unused lifetime or something?

Reply:
Nope :shrug: Good catch. I should do a little scour myself to review these.

Reply:
Tried adding https://rust-lang.github.io/rust-clippy/master/#extra_unused_lifetimes to `lib.rs` and the command line and it seems like it won't pick it up. I wonder if it's a bug in clippy :thinking: